src/main.rs

```diff
@@ -74,7 +96,11 @@ async fn main() -> Result<(), std::io::Error> {
 
     let bind_address = format!("{}:{}", config.listen_address, config.listen_port);
 
-    let data_config = web::Data::new(AppData { config, fetcher });
+    let data_config = web::Data::new(AppData {
+        cache: Mutex::new(TimedCache::with_lifespan(config.cache_lifespan)), // 5min
```

Comment:
So lifespan is global for every cache entry?

Reply:
yes

Reply:
I can create 2 caches for the updater and the game if you want